tests/test_add_sections.py

```diff
@@ -1,11 +1,15 @@
 import re
 from pathlib import Path
 
-from sphinx_testing import with_app
+import pytest
 
 
-@with_app(buildername="html", srcdir="doc_test/add_sections")
-def test_section_is_usable_in_filters(app, status, warning):
+@pytest.mark.parametrize("buildername, srcdir", [("html", "doc_test/add_sections")])
+def test_section_is_usable_in_filters(create_app, buildername):
+    make_app = create_app[0]
+    srcdir = create_app[1]
```

Comment:
Can't we take this one as test_parameter? Or do we even need it here? See next comment about app-init in fixture and not in test